azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageTypeServlet.java

```diff
@@ -74,10 +82,36 @@ protected void handleGet(
       throws ServletException, IOException {
     try {
       final Map<String, String> templateVariableToValue = new HashMap<>();
-      if (IMAGE_TYPE_WITH_ID_URI_TEMPLATE.match(req.getRequestURI(), templateVariableToValue)) {
+      ImageTypeDTO imageTypeDTO = null;
+      if (StringUtils.isNotBlank(req.getPathInfo())) {
```

Comment:
Why request URI check is not needed for this case?

Reply:
My understanding is that req.getRequestURI() will match the entire path, which in this case is variable. IE it would return /imageTypes/imageTypeNameA for GET <server>/imageTypes/imageTypeNameA and /imageType/ImageTypeNameB for GET <server>/imageTypes/imageTypeNameB. I will confirm when I test again after making changes.